shim/debian.compat

```diff
@@ -0,0 +1 @@
+9
```

Comment:
can this be created on the fly ? or do we need to track it?

Reply:
Nope, it can be created.